txp/Pos/Txp/Toil/Failure.hs

```diff
@@ -30,6 +30,8 @@ data ToilVerFailure
     | ToilInvalidInputs ![Text] -- [CSL-814] TODO: make it more informative
     | ToilTooLargeTx { ttltSize  :: !Byte
                      , ttltLimit :: !Byte}
+    | ToilInsufficientFee { tifPolicy :: !TxFeePolicy
+                          , tifSize   :: !Byte }
```

Comment:
I think it's more important to include fee into this error than size. Though size is useful too.